src/explorer/AzureAccountTreeItem.ts

```diff
@@ -2,17 +2,203 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
+import { UiStrings } from "../uiStrings";
+import * as vscode from "vscode";
+import { AzExtParentTreeItem, AzExtTreeItem, registerEvent, GenericTreeItem, ISubscriptionContext } from '@microsoft/vscode-azext-utils';
+import { AzureSubscriptionHelper } from "../azure/azureLogin/subscriptions";
+import { AzureSessionProvider, ReadyAzureSessionProvider, SelectionType, SignInStatus } from "../azure/azureLogin/authTypes";
+import { GeneralUtils } from "../utils/generalUtils";
+import { AzureAuth } from "../azure/azureLogin/azureAuth";
+import { APIMAccountCommandId } from "../constants";
+import { Subscription } from "@azure/arm-resources-subscriptions";
+import { createSubscriptionTreeItem } from "./ApiManagementProvider";
 
-import { ISubscriptionContext } from '@microsoft/vscode-azext-utils';
-import { AzureAccountTreeItemBase } from '@microsoft/vscode-azext-azureutils';
-import { ApiManagementProvider } from './ApiManagementProvider';
+export function createAzureAccountTreeItem(
+    sessionProvider: AzureSessionProvider,
+): AzExtParentTreeItem & { dispose(): unknown } {
+    return new AzureAccountTreeItem(sessionProvider);
+}
 
-export class AzureAccountTreeItem extends AzureAccountTreeItemBase {
-    public constructor(testAccount?: {}) {
-        super(undefined, testAccount);
+export class AzureAccountTreeItem extends AzExtParentTreeItem {
+    private subscriptionTreeItems: AzExtTreeItem[] | undefined;
+    public static contextValue: string = "azureApiManagementAzureAccount";
+    public readonly contextValue: string = AzureAccountTreeItem.contextValue;
```

Comment:
We have 2 same property names. Could we give them different name?

Reply:
this is a common solution, this `AzureAccountTreeItem` just follow the programming standards like other tree Items. and this context is consumed by `package.json` file like field `view/item/context`
Azure Api Center Ext also do this.